acm-mu/abacus
# added confirmation before deleting submissions, users and problems

# Description



- confirmation window shows before users, submissions and problems are deleted
- upon the deletion occurring successfully a toast container shows
- I want to add a follow up issue for someone else to do for displaying a error message

Fixes #72 


## Type of change





# How Has This Been Tested?

- select a user on the table, click delete selected

<img width="100" alt="Screen Shot 2021-10-08 at 1 36 02 PM" src="https://user-images.githubusercontent.com/17347501/136609311-3129f566-9837-4b26-abb1-82b517c1f43d.png">

- select ok, the user should be deleted, and a message should be displayed confirming the deletion occurred

<img width="100" alt="Screen Shot 2021-10-08 at 1 39 15 PM" src="https://user-images.githubusercontent.com/17347501/136609664-a0998720-d3ad-459e-9a67-5f90e23c471e.png">



- [x] Tested individually.


# Checklist:

- [x] My changes do not break any features.
